Athos/TFEzPCLibrary/Library64_common.ezpc

```diff
@@ -296,33 +457,188 @@ def void Conv2DReshapeInput(int32_pl N, int32_pl H, int32_pl W, int32_pl CI, int
    int64_al[N][((H-FH+zPadHLeft+zPadHRight)/strideH)+1][((W-FW+zPadWLeft+zPadWRight)/strideW)+1][CO] outArr 
 *)
 
-def void Conv2DCSF(int32_pl N, int32_pl H, int32_pl W, int32_pl CI, 
+def void Conv2DCSFO(int32_pl N, int32_pl H, int32_pl W, int32_pl CI, 

```

Comment:
Why this? Lets keep it clean no? Lets have 2 functions - one for the original conv2dcsf and one new one for grouped conv. You can rename the former one to conv2d. But then we need to make sure that Porthos and other places are also unaffected. Easier thing to do is just leave the old one and add a new function for Grouped Conv?